pages/bibliographic-references.xml

```diff
@@ -17,113 +17,140 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-    <att>target</att>, as in the following example:</p>
-         <egXML xmlns="http://www.tei-c.org/ns/Examples">
-<bibl>
-  <ptr target="bm:Villa2013Erma"/>
-</bibl>
-</egXML>
-         <p>If you need to add an indication of exact reference, it should be given in a
-    <gi>citedRange</gi> element, as in the following
-    example:</p>
-         <egXML xmlns="http://www.tei-c.org/ns/Examples">
-<bibl>
-  <ptr target="bm:Villa2014Erma"/>
-  <citedRange unit="page">5-13</citedRange>
-</bibl>
-</egXML>
-         <p>
-            <gi>citedRange</gi> can take several values in
-    <att>unit</att> and can be repeated if there is a need
-    to specify a volume, an entry, etc.</p>
-         <p>Please, <hi rendition="simple:bold">do not</hi> use <att>from</att> and
-    <att>to</att> in <gi>citedRange</gi> as
-    this is unnecessary.</p>
-         <p>Please note that lists of references about a manuscript should be
-    kept with the manuscript and bibliography about a text should be
-    kept with the text, etc. No typed bibliography is entered in the
-    record, only pointers to a common bibliographic database.</p>
-         <p>The target should be a uniquely assigned tag in the
-    <ref target="https://www.zotero.org/groups/ethiostudies/items">Zotero
-    Group Bibliography EthioStudies</ref>. If the item you want to cite
-    does not yet have a unique tag, please assign one with this format
-    and using only Latin alphabet letters</p>
-         <cit>
-            <quote>
- bm:AuthorYYYYkeyword
-</quote>
-         </cit>
-            <p>Please make sure this contains <hi rendition="simple:bold">no special characters</hi>, only digits
-               and Latin ASCII characters; no dashes/hyphens/underscores/diacritics. Make the tag as short as possible. To check that this is unique, after saving the
-    record in Zotero, click on the tag and check that only one item has
-    this tag. If not, change the keyword of the tag just assigned.</p>
-         <p>When adding records to the Zotero Group Library, please follow
-    the principles <ref target="zotero">at this
-    page</ref>.</p>
-</div>
-<div type="level2">
-  <p>You can use a <att>corresp</att> to link a <gi>bibl</gi> element to a
-part of the file with an <att>xml:id</att>.</p>
-<egXML xmlns="http://www.tei-c.org/ns/Examples">
-  <bibl corresp="#t3arabic">
-  <ptr target="bm:Enger1856Transitu"/>
-  </bibl>
-</egXML>
-<p>The above example would point the bibliographical reference directly to the <gi>title</gi> with that ID.</p>
-<egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <head>Bibliographic references for enriched records</head>
+            <p>Following <ref target="https://github.com/BetaMasaheft/Documentation/issues/2630">issue
+               2630</ref>, in cases of enriching a historical catalogue description, such as verifying images or
+               adding new information, to acknowledge this, one can include the marked-up by a
```

Comment:
I think there is an English problem in "one can include the marked-up by a", but don't spontaneously see a good correction :)

Reply:
Would "one can include a <gi>bibl</gi> element" work?

Reply:
fine for me!